scripts/geocoder_demo_standalone.py

```
@@ -0,0 +1,311 @@
+    print("• Automatic rate limit coordination")
+    print("• Progress tracking and error recovery")
+    print("• Memory-efficient streaming")
+    
+    # Sample batch results
+    addresses = [
+        "1234 N Highland Ave, Los Angeles, CA",
+        "6801 Hollywood Blvd, Hollywood, CA", 
+        "123 N Beverly Dr, Beverly Hills, CA"
+    ]
+    
+    print(f"\n📋 Sample Batch (3 addresses):")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)